cmd/validators.go

```diff
@@ -130,6 +127,9 @@ func inferArgumentLocation(arg string) common.Location {
 				return common.ELocation.BlobFS()
 			case strings.Contains(host, benchmarkSourceHost):
 				return common.ELocation.Benchmark()
+				// enable targeting an emulator/stack
+			case IPv4Regex.MatchString(host):
+				return common.ELocation.Unknown()
 			}
 
```

Comment:
To confirm my understanding, this just makes the error reporting clearer, and the user can specify their own fromtTo, correct?

Reply:
Yup!